index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>

```

Comment:
This page should also not be in the pull request as it does not comply with the rest of the repo.